src/account_multisig.ts

```diff
@@ -115,11 +115,11 @@ export class AccountMultisig extends Account {
     // helpers
 
     async getRequestNonce(): Promise<Number> {
-        return this.contract.get_request_nonce();
+        return await this.contract.get_request_nonce();
     }
 
     async getRequestIds(): Promise<string> {
-        return this.contract.list_request_ids();
+        return await this.contract.list_request_ids();
```

Comment:
I just want to note that this is the same as removing the `async` and keeping the `return` as it was

Reply:
I know, but I had trouble with that before... perhaps that was JS and with TypeScript it's more clear and safer with typed return values.

Reply:
Sorry, I should have made it clear that I don't much care either way; not sure which is more idiomatic tbh 😄